fc/jira/triage_task.py

```diff
@@ -7,23 +7,68 @@
 
 class TriageTask(Task):
 
-    def __init__(self, a_task: Task):
-        self.title = a_task.title
-        self.description = a_task.description
-        self.id = a_task.id
-        self.url = a_task.url
-        self.type = a_task.type
-        self.state = a_task.state
-        self.auth = a_task.auth
+    # Triage workflow
+    # Triage -> Ready (11)
+    # Ready -> In Progress (21)
+    # In Progress -> Closed (31)
+    # In Progress -> Ready (61)
+    # In Progress -> Blocked (71)
+    # Blocked -> In Progress (81)
+    # Closed -> Triage (41)
+    # Closed -> In Progress (51)
+    transition_dict = {
+        'Open': {
+            None},
```

Comment:
No need to wrap `None` in `{}`.  It can stand on its own.
`'Open': None,`
This applies to others below.

On second thought... do we even need to keep the keys that have no transition?    Why even have `'Open': None` and the others?